src/main/java/io/rebloom/client/Client.java

```diff
@@ -800,10 +800,14 @@ public void tdigestReset(String key) {
   }
 
   @Override
-  public void tdigestMerge(String toKey, String fromKey) {
+  public void tdigestMerge(String toKey, String... fromKey) {
```

Comment:
@sazzad16 can we add the ` [COMPRESSION compression] [OVERRIDE]` options?

Reply:
@filipecosta90 We've added those in Jedis https://github.com/redis/jedis/pull/3097 
My focus here is to keep JRedisBloom _unbroken_, not to include new features i.e. commands, arguments.